tests/network/services.go

```diff
@@ -208,40 +217,36 @@ var _ = SIGDescribe("Services", func() {
 		})
 
 		Context("with a service matching the vmi exposed", func() {
-			var isDualStack bool
-			var ipv6ServiceName string
+			var serviceExpositionManager *vmiServiceExpositionManager
+			var serviceNamePrefix string
 
 			BeforeEach(func() {
-				serviceName = "myservice"
+				serviceNamePrefix = "myservice"
+				serviceExpositionManager = newVMIServiceExpositionManager(inboundVMI, servicePort, serviceNamePrefix, selectorLabelKey, selectorLabelValue)
 
-				service := buildServiceSpec(serviceName, servicePort, servicePort, selectorLabelKey, selectorLabelValue)
-				_, err := virtClient.CoreV1().Services(inboundVMI.Namespace).Create(service)
-				Expect(err).ToNot(HaveOccurred())
+				services := serviceExpositionManager.buildK8sServicesSpec()
+				Expect(services).NotTo(BeEmpty(), "a service should be exposed per each iface. At least one *must* be present.")
 
-				isDualStack = flags.IsDualStackedNetworkConfig
```

Comment:
The flag never existed, it is very strange to see this commit.

Reply:
Busted. This was addressed in a previous commit. Good catch.